src/components/MultiHopTrade/components/TradeInput/components/Claim/hooks/useArbitrumClaimsByStatus.tsx

```diff
@@ -46,71 +47,67 @@ export const useArbitrumClaimsByStatus = (props?: { skip?: boolean }) => {
   const l1Provider = getEthersV5Provider(KnownChainIds.EthereumMainnet)
   const l2Provider = getEthersV5Provider(KnownChainIds.ArbitrumMainnet)
 
-  const queries = useMemo(() => {
-    return {
-      queries: arbitrumWithdrawTxs.map(tx => {
-        return {
-          queryKey: ['claimStatus', { txid: tx.txid }],
-          queryFn: async (): Promise<ClaimStatusResult> => {
-            const receipt = await l2Provider.getTransactionReceipt(tx.txid)
-            const l2Receipt = new ChildTransactionReceipt(receipt)
-            const events = l2Receipt.getChildToParentEvents()
-            const messages = await l2Receipt.getChildToParentMessages(l1Provider)
-            const event = events[0]
-            const message = messages[0]
-            const status = await message.status(l2Provider)
-            const block = (await message.getFirstExecutableBlock(l2Provider))?.toNumber()
-            const timeRemainingSeconds = await (async () => {
-              if (!block) return
-              const latestBlock = await l1Provider.getBlock('latest')
-              const historicalBlock = await l1Provider.getBlock(
-                latestBlock.number - AVERAGE_BLOCK_TIME_BLOCKS,
-              )
-              const averageBlockTimeSeconds =
-                (latestBlock.timestamp - historicalBlock.timestamp) / AVERAGE_BLOCK_TIME_BLOCKS
-              const remainingBlocks = block - latestBlock.number
-              return remainingBlocks * averageBlockTimeSeconds
-            })()
-            return {
-              event,
-              message,
-              status,
-              timeRemainingSeconds,
+  const claimStatuses = useQueries({
```

Comment:
Review me with whitespaces off:

- moved outside of memo since we losed type inference, which was the root cause of our derped invalid shape in the first place
- fixes shape
  - `isEnabled` should be `enabled` 
  - `enabled` and all its frens should live in the same level as individual queries, not in the outer scope
  - `refetchInterval` typing of `latestData` was absolutely invalid - meaning we would always overfetch claims (`latestData` is actually a `Query`, not a `TData` - which, when moving queries directly within `useQuery()` was made obvious by tsc errors, we could even ditch the explicit typings in `refetchInterval` if we wanted to)